cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java

```diff
@@ -480,6 +482,7 @@ protected void configure() {
           bind(QueueClientFactory.class).to(HBaseQueueClientFactory.class).in(Singleton.class);
           bind(QueueAdmin.class).to(HBaseQueueAdmin.class).in(Singleton.class);
           bind(HBaseTableUtil.class).toProvider(HBaseTableUtilFactory.class);
+          bind(DatasetProvider.class).to(DefaultDatasetProvider.class);
```

Comment:
In #4325, DataSetsModules has the binding, but in here, I have it only in StreamAdminModules. Will fix this PR to make it more consistent with #4325.